src/seedu/addressbook/Main.java

```diff
@@ -75,11 +76,25 @@ private void exit() {
         ui.showGoodbyeMessage();
         System.exit(0);
     }
+    
+    /**
+     * Throws exception if user deletes storage file. 
+     */
+    private void checkIfStorageFileDeleted() throws FileNotFoundException {
+    	if (!storage.path.toFile().exists()) { // check if storage file exists
+    		throw new FileNotFoundException("User deleted storage file");
```

Comment:
The error message is too specific. Maybe it was a system failure that deleted the file :)